apollo-router/src/plugins/telemetry/metrics/apollo/studio.rs

```diff
@@ -282,10 +298,15 @@ impl From<FieldStat> for crate::plugins::telemetry::apollo_exporter::proto::repo
         Self {
             return_type: stat.return_type,
             errors_count: stat.errors_count,
-            observed_execution_count: stat.latency.entries,
+            observed_execution_count: stat.latency.entries as u64,
             estimated_execution_count: stat.estimated_execution_count as u64,
             requests_with_errors_count: stat.requests_with_errors_count,
-            latency_count: stat.latency.buckets,
+            latency_count: stat
+                .latency
+                .buckets
+                .iter()
+                .map(|&float| float as i64)
```

Comment:
The fact that this function is where the estimated histograms (which use floats) are converted to integers is probably worth calling out with a comment?

Reply:
Yes, definitely